pilot/pkg/model/service.go

```diff
@@ -479,8 +490,8 @@ type IstioEndpoint struct {
 	// Labels points to the workload or deployment labels.
 	Labels labels.Instance
 
-	// Address is the address of the endpoint, using envoy proto.
-	Address string
+	// Addresses is the address of the endpoint, using envoy proto.
```

Comment:
Please add a LOT of comments on what the semantics of this field is now that there are multiple.

Reply:
+1 - one specific comment I'd like to see is a set of constraints around an endpoints' addresses (e.g. all elements of `Addresses` must have the same metadata)

Reply:
okay, I add more description and 2 constraints in comments.